packages/mui-material/src/Grid2/Grid2.spec.tsx

```diff
@@ -0,0 +1,17 @@
+import { createTheme } from '@mui/material/styles';
+
+// Test presence of keys in Grid2Classes in theme
+createTheme({
+  components: {
+    MuiGrid2: {
+      styleOverrides: {
+        root: {
+          backgroundColor: 'red',
+        },
+        container: {
```

Comment:
@ZeeshanTamboli do you think this test serves the purpose?

Reply:
We can add one property each for the dynamic Grid2 classes (`spacing-xs-`, `wrap-xs-`, `grid-xs-`) and move the test to the createTheme spec [here](https://github.com/mui/material-ui/blob/master/packages/mui-material/src/styles/createTheme.spec.ts#L67), as Grid2 spec should focus on the component, not the theme.